src/Library/demos/Drop Down/main.js

```diff
@@ -0,0 +1,68 @@
+import Gtk from "gi://Gtk";
+import GObject from "gi://GObject";
+import Gio from "gi://Gio";
+
+const drop_down = workbench.builder.get_object("drop_down");
+const advanced_drop_down = workbench.builder.get_object("advanced_drop_down");
+
+drop_down.connect("notify::selected-item", () => {
+  const selected_item = drop_down.selected_item.get_string();
+  console.log(selected_item);
+});
+
+const KeyValuePair = GObject.registerClass(
+  {
+    Properties: {
+      key: GObject.ParamSpec.string(
+        "key",
+        null,
+        null,
+        GObject.ParamFlags.READWRITE,
+        "",
+      ),
+      value: GObject.ParamSpec.string(
+        "value",
+        "Value",
+        "Value",
+        GObject.ParamFlags.READWRITE,
+        "",
+      ),
+    },
+  },
+  class KeyValuePair extends GObject.Object {},
+);
+
+const model = new Gio.ListStore();
```

Comment:
```suggestion
const model = new Gio.ListStore({ item_type: KeyValuePair });
```

Strictly speaking, you can always use `GObject.Object` as the GType/item-type, but you get free type checking for setting it to a specific type.